docs/3.0rc/develop/recover-from-failure.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Recover from failure
+# description: TODO
+---
+
+Prefect can automatically retry tasks on failure. In Prefect, a task _fails_ if
+its Python function raises an exception.
+
+To enable retries, pass `retries` and `retry_delay_seconds` parameters to your
+task. If the task fails, Prefect will retry it up to `retries` times, waiting
+`retry_delay_seconds` seconds between each attempt. If the task fails on the
+final retry, Prefect marks the task as _crashed_ if the task raised an exception
+or _failed_ if it returned a string.
+
+!!! note "Retries don't create new task runs"
```

Comment:
```suggestion
<Note> "Retries don't create new task runs" 
```

Reply:
Mkdocs style admonitions with `!!!` don't work in Mintlify. This one just caught my eye, but I can find/replace them all post merging this PR.